src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/Huffman.cs

```diff
@@ -270,7 +270,7 @@ internal class Huffman
             (0b11111111_11111111_11111111_11111100, 30)
         };
 
-        private static readonly (int codeLength, int[] codes)[] s_decodingTable = new[]
+        private static readonly (int codeLength, int[] codes)[] _decodingTable = new[]
```

Comment:
What are the chances we could collectively agree on the s_ prefix for statics?  When reviewing code, it's really helpful to me to know whether something is instance state or global state.  For example, with "_" if I saw something mutating this table, I'd think "ok, it's probably safe", but with "s_" if I saw something mutating this table, alarm bells would start going off unless I saw the appropriate synchronization in place.

Reply:
I'm not going to fight it; I'm not going to fight for 2 static variable names. However, we should try to unify our coding guidelines as much as possible.

Reply:
Thanks.

Reply:
I'd be ok switching to `s_` in ASP.NET Core.